code/game/objects/items/weapons/twohanded.dm

```diff
@@ -412,3 +412,15 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+/obj/item/weapon/twohanded/rock
+	name = "rock"
+	desc = "If there were two miners were on the mining station and one of them killed the other with a rock, would that be fucked up or what?"
```

Comment:
the "were" between "miners" and "on" is unnessiary and breaks flow with the meme your invoking.

Same for "of them" between "one" and "killed"

if there were two miners on the mining station and one killed the other with a rock, would that be fucked up or what?